### PR TITLE
Add new key value in user lang en

### PR DESCRIPTION
**Pull request by shuvo-asl**

### CODE REVIEW

**Comment by mh-shuvo** [comment]
Roll will be roll number

File: resources/lang/en/user.php [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?php
 return [
-  "name => "Enter your name"  
+  "name => "Name",
+  "roll" => "Roll"
```
Comment (mh-shuvo) [inline resources/lang/en/user.php]: add the comma of end of the 4th line

**Review by mh-shuvo — CHANGES_REQUESTED** [review]

File: resources/lang/en/user.php (hunk 1 above)
Comment (shuvo-asl) [reply resources/lang/en/user.php]: Okay sir

**Review by mh-shuvo — APPROVED** [review]
Good Job @shuvo-asl

**Pull request merged**